move-mutation-test/tests/testsuite.rs

```diff
@@ -0,0 +1,75 @@
+use aptos::common::types::MovePackageDir;
+use log::info;
+use move_mutation_test::cli::CLIOptions;
+use move_mutation_test::cli::TestBuildConfig;
+use move_mutation_test::run_mutation_test;
+use mutator_common::report::Report;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+const RED_ZONE: usize = 128 * 1024; // 128 KiB
+const STACK_SIZE: usize = 32 * RED_ZONE; // 4 MiB
```

Comment:
`RED_ZONE` and `STACK_SIZE` consts are set to 128 KiB and 4 MiB for any particular reason?
I mean why those particular values have been used?

Reply:
An arbitrary choice after doing some testing based on crate docs. I tweaked it a bit, but it doesn't affect the overall execution time.
As long as the stack doesn't overflow anymore, it does the trick.

Reply:
I might tweak it a bit more during the next testing phase (even add a comment here)